--- file-roller-43.0/data/packages.match.orig	2023-05-18 22:57:11.319556933 +0200
+++ file-roller-43.0/data/packages.match	2023-05-18 22:59:06.448933223 +0200
@@ -1,20 +1,20 @@
 [Package Matches]
-7zip=
-7zip-full=
-7zip-rar=
+7zip=p7zip
+7zip-full=p7zip
+7zip-rar=p7zip
 arj=
 binutils=
 brotli=
 bzip2=
 cpio=
 dpkg=
-genisoimage=
+genisoimage=cdrkit-mkisofs
 gzip=
 lha=
 lrzip=
 lz4=
 lzip=
-lzma=
+lzma=xz
 lzop=
 ncompress=
 rar=
