--- file-roller-3.6.3/src/fr-archive.c.orig	2012-11-25 12:53:27.000000000 +0100
+++ file-roller-3.6.3/src/fr-archive.c	2013-01-06 17:29:09.217910387 +0100
@@ -28,9 +28,6 @@
 #include <glib.h>
 #include <glib/gi18n.h>
 #include <gio/gio.h>
-#if ENABLE_MAGIC
-#  include <magic.h>
-#endif
 #include "glib-utils.h"
 #include "file-utils.h"
 #include "gio-utils.h"
--- file-roller-3.6.3/src/glib-utils.c.orig	2012-09-28 11:19:18.000000000 +0200
+++ file-roller-3.6.3/src/glib-utils.c	2013-01-06 17:29:09.834605567 +0100
@@ -19,9 +19,13 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <config.h>
 #include <ctype.h>
 #include <string.h>
 #include <stdio.h>
+#if ENABLE_MAGIC
+#  include <magic.h>
+#endif
 #include <glib.h>
 #include <glib/gi18n.h>
 #include <glib/gprintf.h>
