--- file-roller-2.16.3/data/file-roller.desktop.in.in~	2007-01-29 22:05:13.000000000 +0100
+++ file-roller-2.16.3/data/file-roller.desktop.in.in	2007-02-10 18:00:56.000000000 +0100
@@ -8,8 +8,8 @@
 Terminal=false
 Type=Application
 Icon=file-roller
-Categories=GTK;GNOME;Application;Utility;
-MimeType=application/x-ar;application/x-arj;application/x-bzip;application/x-bzip-compressed-tar;application/x-compress;application/x-compressed-tar;application/x-deb;application/x-gtar;application/x-gzip;application/x-lha;application/x-lhz;application/x-rar;application/x-rar-compressed;application/x-tar;application/x-zip;application/x-zip-compressed;application/zip;multipart/x-zip;application/x-rpm;application/x-jar;application/x-war;application/x-ear;application/x-java-archive;application/x-lzop;application/x-zoo;application/x-cd-image;application/x-7z-compressed;application/x-gzpostscript;
+Categories=GTK;GNOME;Utility;Archiving;
+MimeType=application/x-ar;application/x-arj;application/x-bzip;application/x-bzip2;application/x-bzip-compressed-tar;application/x-bzip2-compressed-tar;application/x-compress;application/x-compressed-tar;application/x-deb;application/x-gtar;application/x-gzip;application/x-lha;application/x-lhz;application/x-rar;application/x-rar-compressed;application/x-tar;application/x-zip;application/x-zip-compressed;application/zip;multipart/x-zip;application/x-rpm;application/x-jar;application/x-war;application/x-ear;application/x-java-archive;application/x-lzop;application/x-zoo;application/x-cd-image;application/x-7z-compressed;application/x-gzpostscript;
 X-GNOME-DocPath=file-roller/file-roller.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=file-roller
