diff -urN file-roller-2.6.1.orig/configure.in file-roller-2.6.1/configure.in
--- file-roller-2.6.1.orig/configure.in	2004-04-19 23:18:43.232238432 +0200
+++ file-roller-2.6.1/configure.in	2004-04-19 23:18:54.781482680 +0200
@@ -55,7 +55,7 @@
 
 dnl ******************************
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko lt lv mk ml mn ms nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta th tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta th tr uk vi zh_CN zh_TW"
 
 GETTEXT_PACKAGE=file-roller
 AC_SUBST(GETTEXT_PACKAGE)
